runtime/compiler/codegen/J9RecognizedMethodsEnum.hpp

```diff
@@ -156,6 +156,16 @@
    java_lang_Double_init,
    java_lang_String_init,
 
+   // Unboxing methods
+   java_lang_Boolean_booleanValue,
+   java_lang_Byte_byteValue,
+   java_lang_Character_charValue,
+   java_lang_Short_shortValue,
+   java_lang_Integer_intValue,
+   java_lang_Long_longValue,
+   java_lang_Float_floatValue,
+   java_lang_Double_doubleValue,
+
```

Comment:
The comment that appears just below talks about recognized constructors for the immutable classes that are declared just above here.  I would suggest moving these various value getter declarations beneath those comments to avoid separating the comments from the constructor enum declarations.